src/assets/controllers/firebaseShortcuts.js

```diff
@@ -0,0 +1,45 @@
+import {firebase} from '@react-native-firebase/firestore';
+
+const getUserByUID = (uid, callback) => {
+  firebase.firestore().collection('Users').where('useruid', '==', uid).get().then((querySnapshot) => {
+    querySnapshot.forEach(snapshot => {
+      let data = snapshot.data();
+      callback(data);
+    });
+  });
+};
+
+const getCollection = collection => {
+  return firebase.firestore().collection(collection);
+};
+
+const getDoc = (collection, document) => {
+  return getCollection(collection).doc(document);
+}
+
+const handleThen = (req, callback) => {
+  return req.catch(e => {
+    console.error(e);
```

Comment:
Don't put console.log(), when you push your code from GitHub. Instead you can make use of Alert.alert()

Reply:
done